src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -415,6 +445,57 @@ private function getAbsoluteFilePath(string $filePath): string
         return sprintf('%s/%s', $this->getStubsDirectory(), $filePath);
     }
 
+    /**
+     * @param list<Node\Name> $nameNodes
+     *
+     * @return list<Node\Name>
+     */
+    private function modifyExtendsImplementsByPhpVersion(string $className, array $nameNodes): array
+    {
+        $modifiedNames = [];
+        foreach ($nameNodes as $nameNode) {
+            $name = $nameNode->toString();
+
+            if ($className === ParseError::class) {
```

Comment:
Is there a way to write this in a more "linear" version? Most of these `if/else` blocks seem nested and related to each other :thinking: 

Perhaps `continue;` when it does **not** match any of these 3? :thinking:

Reply:
I've really tried but I don't know how to improve it. The stubs are sometimes really garbage.

Reply:
I totally trust you on that, thanks for putting ij the effort 👍